services/metadactyl-clj/src/metadactyl/routes/admin.clj

```diff
@@ -95,29 +93,29 @@
           :summary "Update App Documentation"
           :description "This service is used by DE administrators to update documentation for a single
           App"
-          (service/coerced-trap uri AppDocumentation
-                                apps/admin-edit-app-docs current-user app-id body))
+          (coerce! AppDocumentation
+                   (apps/admin-edit-app-docs current-user app-id body)))
 
-  (POST* "/:app-id/documentation" [:as {uri :uri body :body}]
+  (POST* "/:app-id/documentation" []
          :path-params [app-id :- AppIdPathParam]
          :query [params SecuredQueryParams]
          :body [body (describe AppDocumentationRequest "The App Documentation Request.")]
          :return AppDocumentation
          :summary "Add App Documentation"
          :description "This service is used by DE administrators to add documentation for a single App"
-         (service/coerced-trap uri AppDocumentation
-                               apps/admin-add-app-docs current-user app-id body)))
+         (coerce! AppDocumentation
+                  (apps/admin-add-app-docs current-user app-id body))))
```

Comment:
Since `service/coerced-trap` was wrapping the response in `success-response` or `ok`, I think these `coerce!` calls still need to be wrapped in an `ok`.